anti.md

```diff
@@ -58,12 +98,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+You'd have to spend the whole time on the language part because there is no
+systems aspect except following the instructinos on your list.
```

Comment:
That's a good point.  Assuming we don't have laundry bots or anything, the executable would be steps that a person has to go through.